layouts/partials/tw/donate-prefill-lg.html

```diff
@@ -57,7 +57,7 @@
         m: 'monthly',
         y: 'yearly',
       }[this.freq];
-      return `https://checkout.fundjournalism.org/memberform?org_id=spotlightpa&campaign=${campaign}&theme=${theme}&amount=${this.amount}&installmentPeriod=${period}&utm_source=spotlightpa.org&utm_medium=elections&utm_campaign=donate-prefill`;
+      return `https://checkout.fundjournalism.org/memberform?org_id=spotlightpa&campaign={{ $donateCampaign }}&theme={{ $donateTheme }}&amount=${this.amount}&installmentPeriod=${period}&utm_source=spotlightpa.org&utm_medium=elections&utm_campaign=donate-prefill`;
```

Comment:
I had a suggestion for how to change this on a different PR. Let's just merge this one first though so there's no merge conflict.

Reply:
I've made a note of this and will roll into the other pr